.buildkite/wsl2-mirrored.yml

```diff
@@ -0,0 +1,17 @@
+# WSL2 Mirrored Mode networking with docker-ce (docker native inside WSL2 == Docker inside)
```

Comment:
I see Docker Desktop installed on `tb-wsl-mirrored-15-1`, if it is not needed then it is better to uninstall it?

Reply:
I uninstalled docker desktop on tb-wsl-mirrored-15